### ISSUE TITLE
Unable to reproduce tutorial on latitudinally, seasonally varying RCE with CAM3 radiation

### ISSUE THREAD

**Issue by andrewpauling** [brian-rose/climlab#107]
I have been trying to reproduce the tutorial on RCE with the CAM3 radiation scheme, specifically with latitudinally and seasonally varying RCE. The first part with no meridional diffusion gives a temperature section that looks just like the example, but when I add in the meridional diffusion subprocess and integrate forward there is barely any difference from the first temperature profile (a few Kelvin at most). Has something changed since that tutorial was made? Thanks!

**Comment by brian-rose** [comment]
Yes, there were some changes to the diffusion modules that unfortunately broke some older examples that apply diffusion on 2D domains. This is almost surely the same problem as in https://github.com/brian-rose/ClimateModeling_courseware/issues/5.

You can see in these notes (https://github.com/brian-rose/ClimateModeling_courseware/blob/master/Lectures/Lecture22%20--%20Water%2C%20water%20everywhere!.ipynb) there is currently a hack to work around this bug. This code basically just redefines the diffusion module back to what it used to be.

We'll fix these examples properly in the next climlab release.

Thanks for bringing this to my attention!

**Comment by brian-rose** [comment]
FYI, in general the notebooks in https://github.com/brian-rose/ClimateModeling_courseware tend to be more up-to-date -- they get a lot more attention as they are actively used in courses at my university.

**Comment by andrewpauling** [comment]
Great thanks for the fix and the extra resources!
